blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipeline.java

```diff
@@ -11,6 +11,7 @@
 import org.kohsuke.stapler.json.JsonBody;
```

Comment:
@vivek this was the major change. Most places in the codebase that need `BlueOrganization` references already have a `BluePipeline` in scope.